doc/source/io.rst

```diff
@@ -2059,6 +2064,11 @@ using Hadoop or Spark.
   df
   df.to_json(orient='records', lines=True)
 
+  # chunksize has no effect when reading a string.
```

Comment:
this is an odd comment here
it does work and does iterate; you are referring to memory savings which is not relevant here

Reply:
You're right. @chris-b1 commented on the same line, I was waiting for a response from him before changing it. Do you agree that we should support chunksize when a string is passed? Either way I will change the documentation.

Reply:
I didn't know we would fall back to parsing the string if not a file (arguably shouldn't, but whole other issue - xref #5924)

So I'm fine not supporting chunksize with a string, but would like this comment to be clearer, or removed.